src/modules/stakewise/stakewise-daemon/server/status/get-validator-statuses.go

```diff
@@ -62,28 +63,33 @@ func (c *statusGetValidatorsStatusesContext) PrepareData(data *swapi.ValidatorSt
 	if err != nil {
 		return fmt.Errorf("error getting public keys: %w", err)
 	}
-	statuses, err := bc.GetValidatorStatuses(context.Background(), publicKeys, nil)
+	statusResponse, err := bc.GetValidatorStatuses(context.Background(), publicKeys, nil)
 	if err != nil {
 		return fmt.Errorf("error getting validator statuses: %w", err)
 	}
 
+	registeredPubkeys := make([]beacon.ValidatorPubkey, 0)
+	for _, pubkeyStatus := range nodesetStatusResponse {
+		registeredPubkeys = append(registeredPubkeys, pubkeyStatus.Pubkey)
+	}
+
 	beaconStatuses := make(map[string]types.ValidatorState)
```

Comment:
You know, this kind of bugged me last time we were here and now that I think about it, it's probably worth mentioning: maps in `go` don't preserve their order, so when you enumerate them, the order of the elements you get is random. That means when you send this back to the CLI and it prints the elements, the order it prints them in could be different each time. For posterity it's probably best to preserve their order in either chronological or lexicographical (based on pubkey) form instead of randomizing, which means this'd be a slice of a wrapper struct with the details instead of a map.

@VVander what do you think?